README.md

```diff
@@ -87,32 +87,97 @@ If you want different passwords, database names or ports you can change them
+            Container(oauth, "OAuth2-Proxy", "Authentication for REST-API")
+            Container(validator, "CSAF validator service", "node")
+
+            Container_Boundary(c1, "Backend") {
+                Container(backend, "CSAF-CMS-Backend", "Spring Boot")
+                ContainerDb(backend-db, "CouchDB", "CMS-Backend-Database")
+            }
+        }
+    }
+
+    Rel(user, reverseproxy,"","HTTPS")
+    Rel(reverseproxy, secvisogram,"/")
+    Rel(reverseproxy, oauth,"/api/*")
+    Rel(reverseproxy, keycloak,"/realm/csaf/")
+    Rel(oauth, validator, "/api/v1/test")
+    Rel(oauth, validator, "/api/v1/validate")
+    Rel(oauth, backend, "/api/v1/advisories/*")
+    Rel(backend, backend-db,"")
+    Rel(backend, keycloak,"")
+    Rel(keycloak, keycloak-db,"")
+   
+
+```
+
 - run `docker compose up`
 - After Keycloak is up, open a second terminal window and run
   `docker compose up csaf-keycloak-cli` to import a realm with all the users
   and roles already set up.
 - To set up our CouchDB server open `http://127.0.0.1:5984/_utils/#/setup`
-  and run the [Single Node Setup](https://docs.couchdb.org/en/stable/setup/single-node.html). This creates databases like **_users** and
-  stops CouchDB from spamming our logs
+  and run the [Single Node Setup](https://docs.couchdb.org/en/stable/setup/single-node.html). This creates databases like **_users** and stops CouchDB from spamming our logs (Admin credentials from .env)
+- Create a database in CouchDB with the name specified in `CSAF_COUCHDB_DBNAME`
 - Open `http://localhost:9000/` and log in with the admin user.
     - The port is defined in .env - CSAF_KEYCLOAK_PORT, default 9000
+    - Select `CSAF`-Realm
     - On the left side, navigate to "Clients" and select the Secvisogram client.
     - Select the **Credentials** tab and copy the Secret. This is our
       `CSAF_CLIENT_SECRET` environment variable.
 - [Generate a cookie secret](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/#generating-a-cookie-secret)
   and paste it in `CSAF_COOKIE_SECRET`.
-- Create a database in CouchDB with the name specified in `CSAF_COUCHDB_DBNAME`
 - restart compose
 - (required for exports) install [pandoc (tested with version 2.18)](https://pandoc.org/installing.html)
   as well as [weasyprint (tested with version 56.0)](https://weasyprint.org/) and make sure both are in
   your PATH
 - (optional for exports) define the path to a company logo that should be used in the exports through the environment variable `CSAF_COMPANY_LOGO_PATH`. The path can either be relative to the project root or absolute. See .env.example file for an example.
 
 You should now be able to start the spring boot application, navigate to
-`localhost:4180/api/v1/about`, log in with one of the users and get a
+`http://localhost/api/v1/about`, log in with one of the users and get a
 response from the server.
 The port is defined in .env - CSAF_APP_EXTERNAL_PORT, default 4180
```

Comment:
This seems confusion, as the `CSAF_APP_EXTERNAL_PORT` is not longer relevant for the URL as we use the reverse proxy. Maybe, we should remove that line or explicitly mention that it is only needed for internal communications in docker? :thinking:

Reply:
> The port is defined in .env - CSAF_APP_EXTERNAL_PORT, default 4180

remove in commit [f2017a1](https://github.com/secvisogram/csaf-cms-backend/pull/195/commits/f2017a17a361bd5228347998345bc6a611bfcad5)